stable/syncthing/2.1.0/test_values.yaml

```diff
@@ -0,0 +1,36 @@
+# Default values for Syncthing.
+
+image:
+  repository: syncthing/syncthing
+  pullPolicy: IfNotPresent
+  tag: 1.15
```

Comment:
Be sure this Image runs safely as non-root!

Considering it already passed a test, I think it does... Just verify all features work ;-)

Reply:
I'm personally running it as non root and everything seems to work fine, I'm also not aware of any features in Syncthing that would require running as root.